en/publish/publish-wechatgame-sub-domain.md

```diff
@@ -48,79 +48,80 @@ Window update
+**Note**: The **Title** in the **Build** panel must match the name of the **Open Data Field Code Directory** set in the main context project.
 
-**Note**: **Game Name** must match the [Open Data Field Code Directory] name set in the main domain project.
 
 ![](./publish-wechatgame/open-data-project-build.png)
 
-Or you can modify the **release path** to manually copy the release package to the release package directory of the primary domain project after the open data domain project is built. As shown below:
+Or you can not modify the **Build Path** to manually copy the release package to the release package directory of the main context project after the open data context project is built. As shown below:
 
 ![](./publish-wechatgame/package.png)
 
-4. Click [Run] in the main domain project to launch the WeChat Developer tool, and then publish and debug according to the normal process of the previous WeChat game.
+4. Click **Run** in the main context project to launch the WeChat Mini Game Developer tool, and then publish and debug according to the normal process of the previous WeChat game.
 
 ![](./publish-wechatgame/preview.png)
 
-**Note: If you publish the open data domain domain and then publish the primary domain, the release code of the open data domain will be overwritten, and we will optimize the experience in subsequent versions**
+**Note: If you publish the open data context domain and then publish the primary context, the release code of the open data context will be overwritten, and we will optimize the experience in subsequent versions**
 
 ## Reference link
 
-The [Open Data Domain Sample Project] (https://github.com/cocos-creator/demo-wechat-subdomain/archive/master.zip) provided by Cocos Creator contains examples of the use of WeChat open data fields.
+The [Open Data Context Sample Project](https://github.com/cocos-creator/demo-wechat-subdomain/archive/master.zip) provided by __Cocos Creator__ contains examples of the use of WeChat open data fields.
 
-[WeChat official document: Guide to the use of relational chain data] (https://developers.weixin.qq.com/minigame/dev/tutorial/open-ability/open-data.html)
+[WeChat official document: Relationship Chain Data Usage Guide](https://developers.weixin.qq.com/minigame/dev/tutorial/open-ability/open-data.html)
 
 ---------------------
 
-#旧版本开放数据域发布
+# Publish to the Open Data Context (version before update)
 
-The following method is applicable to v2.0.0. For earlier versions, please refer to [Accessing Open Data Domain of WeChat Game] (http://docs.cocos.com/creator/1.10/manual/en/publish/publish-wechatgame. Html#%E6%8E%A5%E5%85%A5%E5%BE%AE%E4%BF%A1%E5%B0%8F%E6%B8%B8%E6%88%8F%E7%9A%84 %E5%BC%80%E6%94%BE%E6%95%B0%E6%8D%AE%E5%9F%9F)
+The following method is applicable to v2.0.0. For earlier versions, please refer to [Accessing Open Data Domain of WeChat Game](http://docs.cocos2d-x.org/creator/1.10/manual/en/publish/publish-wechatgame.html)
```

Comment:
oh, i meant to ask you about this because the chinese doc does the same thing. fixed in both places.
![screenshot from 2018-12-04 08-46-56](https://user-images.githubusercontent.com/7491758/49446077-586af500-f7a1-11e8-89b2-b28d69f9bac0.png)

Reply:
Sorry, the Chinese document is wrong. I will modify it as soon as I have time.